Simple-Projects/Simple_Projects/Basic/Complex/Complex.cpp

```diff
@@ -1,10 +1,10 @@
+#include <ostream>
```

Comment:
Note how it's not iostream, but just ostream - we don't need cin or cout (or cerr and clog), and we don't need the istream definition, either. Use what's needed and don't pay (with compilation time) for more